project/AppDependencies.scala

```diff
@@ -3,54 +3,41 @@ import play.sbt.PlayImport.caffeine
 import sbt._
 
 object AppDependencies {
-  val akkaVersion = "2.6.14"
-  val bootstrapPlayVersion = "5.10.0"
+  val akkaVersion = "2.6.19"
```

Comment:
Pinning the Akka version was necessary when using simple-reactivemongo. You _shouldn't_ need to do it with hmrc-mongo